packages/cozy-mespapiers-lib/src/components/SearchResult/useActions.jsx

```diff
@@ -17,49 +18,64 @@ import {
   editContact,
   copyReminderContent,
   forward,
+  forwardTo,
   download
 } from '../Actions/Items'
 import { useModal } from '../Hooks/useModal'
 import { useMultiSelection } from '../Hooks/useMultiSelection'
 
-const useActions = doc => {
+const useActions = (docs, { isActionBar, actionsOptions } = {}) => {
   const webviewIntent = useWebviewIntent()
+  const { t, f } = useI18n()
   const { pushModal, popModal } = useModal()
   const { addMultiSelectionFile } = useMultiSelection()
   const [isPrintAvailable, setIsPrintAvailable] = useState(false)
 
-  const isNoteDoc = isFile(doc) ? isNote(doc) : false
-  const isPDFDoc = doc.mime === 'application/pdf'
+  const hasNoteDoc =
+    docs.length > 0 && docs.every(doc => isFile(doc))
+      ? docs.length > 0 && docs.every(doc => isNote(doc))
+      : false
+
+  const isPDFDoc = docs[0]?.mime === 'application/pdf'
 
   const actions = useMemo(
     () =>
       makeActions(
         [
-          isNoteDoc && copyReminderContent,
-          select,
-          divider,
-          forward,
+          !isActionBar && hasNoteDoc && copyReminderContent,
+          !isActionBar && select,
+          !isActionBar && divider,
+          !isActionBar && forward,
+          isActionBar && forwardTo,
           download,
-          isPrintAvailable && isPDFDoc && print,
-          open,
-          divider,
-          rename,
-          isNoteDoc && editContact,
-          divider,
-          viewInDrive,
-          divider,
+          isPrintAvailable && docs.length === 1 && isPDFDoc && print,
+          !isActionBar && open,
+          !isActionBar && divider,
+          !isActionBar && rename,
+          !isActionBar && hasNoteDoc && editContact,
+          !isActionBar && divider,
+          !isActionBar && viewInDrive,
+          !isActionBar && divider,
```

Comment:
nit: On pourrait peut-être sortir ce tableau pour le conditionner une seul fois sur l'`ActionBar`

Reply:
pas si simple, le principe d'utiliser le hook useActions c'est que c'est le hook qui a l'intelligence, et côté appel il n'y a pas de logique, on ne fait que utiliser le hook. Ceci afin d'avoir toute la logique au même endroit et assurer une homogénéité dans les actions alors qu'on s'en sert à différents endroits. L'ActionsBar est une exception à la règle (jusqu'à maintenant les actions ne concernaient que des ActionsMenu). A voir dans une prochaine PR...